src/diffusers/models/attention_processor.py

```diff
@@ -56,7 +56,8 @@ def __init__(
         bias=False,
         upcast_attention: bool = False,
         upcast_softmax: bool = False,
-        cross_attention_norm: bool = False,
+        cross_attention_norm: Optional[str] = None,
```

Comment:
This can be safely upgraded to an `Optional[str]` as we only ever use it internally through the "K attention" blocks which are updated in this PR